lib/datasource/git-tags/index.js

```diff
@@ -0,0 +1,22 @@
+const { getRemoteInfo } = require('isomorphic-git');
+
+async function getPkgReleases({ lookupName }) {
+  try {
+    const info = await getRemoteInfo({
+      url: lookupName,
```

Comment:
how efficient have you found this vs a raw `git ls-remote --tags` ?

Reply:
Thought I've made overkill by using lib instead of just calling child process. But wow:

![image](https://user-images.githubusercontent.com/1239644/58980365-35b5a980-87e1-11e9-9c27-3ef9f2f9f91c.png) 
(via spawn)

![image](https://user-images.githubusercontent.com/1239644/58980450-6dbcec80-87e1-11e9-860b-2d4980a44caf.png)
(via `isomorphic-git`)

I've quickly commented mocks and expectations, so it actually was same request 3 times.

Reply:
My bad, i've forget about mocking :)
But the library is about 200ms faster anyway

Reply:
At the another hand, don't know if it's reasonable to introduce new library with its deps. But they're using renovate :)